scripts/install-deps.sh

```diff
@@ -44,15 +44,24 @@ echo "Mmmmm this ${FLAVOR} flavor is so delcious"
 
 # Do stuff on each distro
 case $FLAVOR in
+  arch)
+    pacman -Syu ruby
+    gem install rdoc fpm
+    ;;
   debian)
     apt-get update && apt-get -y --force-yes install \
+      bsdtar \
       build-essential \
       curl \
       file \
-      rpm
+      rpm \
+      ruby
+      gem install rdoc fpm
     ;;
   fedora)
-    echo "Not implemented yet!"
+  redhat)
```

Comment:
Broken for me
`
stderr: ./scripts/install-deps.sh: line 62: syntax error near unexpected token `)'
./scripts/install-deps.sh: line 62: `  redhat)'
`

Reply:
Further more should fedora support be in another PR?

Reply:
@inverse: I appreciate the feedback.  Yeah, this should be separated out, although redhat and fedora should have identical sections (in theory, at least).